pygccxml/parser/scanner.py

```diff
@@ -370,14 +426,19 @@ def __read_namespace(self, attrs):
             # that is almost true: gcc mangale name using top file name.
             # almost all files has '.' in name
             ns_name = ''
-        return self.__decl_factory.create_namespace(name=ns_name)
+        decl = self.__decl_factory.create_namespace(name=ns_name)
+        if attrs.get(XML_AN_COMMENT):
```

Comment:
I think I prefer ` if attrs.get(XML_AN_COMMENT) is not None` for the style here, and everywhere where `.get` was used below and in the rest of the changes from this PR.

I think it brings more clarity on the intent here. Explicit is better than implicit.